invenio_rdm_records/stats/templates/events/record_view/v7/record-view-v1.0.0.json

```diff
@@ -0,0 +1,70 @@
+{
+  "index_patterns": ["__SEARCH_INDEX_PREFIX__events-stats-record-view-*"],
+  "settings": {
+    "index": {
+      "refresh_interval": "5s"
+    }
+  },
+  "mappings": {
+    "date_detection": false,
+    "dynamic": false,
```

Comment:
why `false` and not `strict`?

With `false` new fields are ignored. These fields will not be indexed or searchable, but will still appear in the `_source` field of returned hits. These fields will not be added to the mapping, and new fields must be added explicitly.
With `strict` If new fields are detected, an exception is thrown and the document is rejected. New fields must be explicitly aded to the mapping.